### ISSUE TITLE
[Feature Request] include flutter.jar

### ISSUE THREAD

**Issue by liqwik** [leoafarias/fvm#393]
I'm trying to develop the flutter plugin in android studio, but it seems like the android studio didn't recognize the flutter module. It's required the flutter.jar to include into the project.

<img width="582" alt="Screen Shot 2022-02-26 at 11 29 10 AM" src="https://user-images.githubusercontent.com/42688967/155828923-1f959cc4-6107-4199-981b-dd03922f77fe.png">
d

**Comment by leoafarias** [comment]
@liqwik sorry but I am not sure I understand the issue. Was Flutter sdk configured from FVM as described in the docs?

**Comment by liqwik** [comment]
Yes, I have followed the document for the configuration, and everything work well. 

Recently, I develop the plugin and not able to run in Android as you can see above picture. It is kind of unknown the flutter in android studio. 

After google for a while, they suggest me to copy the flutter.jar into the project, but I can't find it in the FVM. 

I thought the FVM might be not include it in the fvm directory. 

Anyway, until now I have fixed it by open the example/android directory instead of the android/ directory, everything work fine.

**Comment by leoafarias** [comment]
Fvm creates a symlink to a cache directory from the project.. use “fvm list” and will give you the path

**Comment by liqwik** [comment]
Thank you for your support. Anyway, It is fine since I can work on the Android Plugin without the flutter.jar file anymore.
